daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -35,10 +35,25 @@ const serverHost = "127.0.0.1"; // App Host of this Example Server
 const serverPort = "50051"; // App Port of this Example Server
 
 // HTTP Example
-const server = new DaprServer(serverHost, serverPort, daprHost, daprPort);
+const server = new DaprServer({
+  serverHost,
+  serverPort,
+  clientOptions: {
+    daprHost,
+    daprPort,
+  },
```

Comment:
I was thinking that we can just show clientOptions once, when we show users how to initialize `DaprServer`, later if we are using 127.0.0.1 and 3500 port, we can skip passing clientOptions. Same for `serverHost` if we are using the default.

Reply:
Leaving that as it is, till some portion of Server docs don't specify defaults for these attributes.
And, docs fixes are planned for v3.2.0